_includes/configuration/advtable-value-series.md

```diff
@@ -0,0 +1,83 @@
+## `advtable_value_series`
+{{ site.requires_5_7 }}
+
+{% if pluginname != "Advanced Tables" %}
+> **Note**: The `advtable_value_series` option requires the Advanced Tables plugin.
+{% endif %}
+
+The `advtable_value_series` option is used in conjuction with the [mceTableToggleSeries]({{site.baseurl}}/plugins/premium/advtable/#commands) command.
+
+The `advtable_value_series` option allows values series to be configured that can be used to populate cells in a table.
```

Comment:
The `advtable_value_series` option configures the incremental values in a series that will be used to populate cells in a table